src/inference/include/openvino/runtime/intel_myriad/hddl_properties.hpp

```diff
@@ -0,0 +1,158 @@
+
+/**
+ * @brief  Metric to get a std::vector<float> of device utilization, String value is METRIC_HDDL_DEVICE_UTILIZATION
+ */
+static constexpr Property<std::vector<float>, PropertyMutability::RO> deviceUtilization{"HDDL_DEVICE_UTILIZATION"};
+
+/**
+ * @brief  Metric to get a std::vector<std::string> of stream ids, String value is METRIC_HDDL_DEVICE_STREAM_ID
+ */
+static constexpr Property<std::vector<std::string>, PropertyMutability::RO> streamId{"HDDL_STREAM_ID"};
+
+/**
+ * @brief  Metric to get a std::vector<std::string> of device tags, String value is METRIC_HDDL_DEVICE_TAG
+ */
+static constexpr Property<std::vector<std::string>, PropertyMutability::RO> deviceTag{"HDDL_DEVICE_TAG"};
+
+/**
+ * @brief  Metric to get a std::vector<int> of group ids, String value is METRIC_HDDL_GROUP_ID
+ */
+static constexpr Property<std::vector<int>, PropertyMutability::RO> groupId{"HDDL_GROUP_ID"};
+
+/**
+ * @brief  Metric to get a int number of device be using for group, String value is METRIC_HDDL_DEVICE_GROUP_USING_NUM
+ */
+static constexpr Property<int, PropertyMutability::RO> deviceGroupUsingNum{"HDDL_DEVICE_GROUP_USING_NUM"};
+
+/**
+ * @brief  Metric to get a int number of total device, String value is METRIC_HDDL_DEVICE_TOTAL_NUM
+ */
+static constexpr Property<int, PropertyMutability::RO> deviceTotalNum{"HDDL_DEVICE_TOTAL_NUM"};
+
+//    RW properties
+
+/**
+ * @brief [Only for HDDLPlugin]
+ * Type: Arbitrary non-empty string. If empty (""), equals no set, default: "";
+ * This config makes the executable networks to be allocated on one certain device (instead of multiple devices).
+ * And all inference through this executable network, will be done on this device.
+ * Note: Only one network would be allocated to one device.
+ * The number of devices which will be used for stream-affinity must be specified in hddl_service.config file.
+ * Example:
+ * "service_settings":
+ * {
+ *     "stream_device_number":5
+ * }
+ * It means that 5 device will be used for stream-affinity
+ */
+static constexpr Property<std::string, PropertyMutability::RW> setStreamId{"HDDL_SET_STREAM_ID"};
+
+/**
+ * @brief [Only for HDDLPlugin]
+ * Type: Arbitrary non-empty string. If empty (""), equals no set, default: "";
+ * This config allows user to control device flexibly. This config gives a "tag" for a certain device while
+ * allocating a network to it. Afterward, user can allocating/deallocating networks to this device with this "tag".
+ * Devices used for such use case is controlled by a so-called "Bypass Scheduler" in HDDL backend, and the number
+ * of such device need to be specified in hddl_service.config file.
+ * Example:
+ * "service_settings":
+ * {
+ *     "bypass_device_number": 5
+ * }
+ * It means that 5 device will be used for Bypass scheduler.
+ */
+static constexpr Property<std::string, PropertyMutability::RW> setDeviceTag{"HDDL_SET_DEVICE_TAG"};
+
+/**
+ * @brief [Only for HDDLPlugin]
+ * Type: "YES/NO", default is "NO".
```

Comment:
Please fix documentation for `bool` types.

Reply:
Fixed